Dockerfile

```diff
@@ -55,6 +55,11 @@ RUN apt-get update -y && apt-get -y upgrade && apt-get install --no-install-reco
     apt-get purge --auto-remove && \
     apt-get clean
 
+RUN apt-get update && apt install -qqy gnupg2 wget unzip libnss3-dev libgdk-pixbuf2.0-dev libgtk-3-dev libxss-dev && \
```

Comment:
This docker file is optimized for production, we should not be adding test artifacts to it (these will bloat it, yet they are not needed for the runtime).

Perhaps we should create another one that uses this as the base